.github/workflows/workflow.yml

```diff
@@ -24,7 +24,11 @@ jobs:
       - run: npm ci
       - run: npm test
       - run: npm ci
-      - run: npm --depth 9999 update
+      - run: npm update
```

Comment:
How is this then working in `passport-saml` if it isn't working here in `node-saml`?

Reply:
I tried running these commands in the same sequence with passport-saml's package.json on this repo, but still the command was failing for this repo and not for passport-saml.